forc-pkg/src/pkg.rs

```diff
@@ -309,6 +309,10 @@ pub struct BuildOpts {
     pub error_on_warnings: bool,
     /// Include all test functions within the build.
     pub tests: bool,
+    /// Whether or not to generate output logs for forc test.
```

Comment:
```suggestion
    /// Whether or not to generate output logs for forc test.
```

I think we can improve the wording here because this flag does not alter the behviour of logs for contracts etc right? 

We should probably say that this flag is enabling logs in predicates or something like that maybe?

Reply:
addressed: https://github.com/FuelLabs/sway/pull/6996/commits/54eb60dee3120de982f85a91a2f4e6dabd4cfa6b
Feel free to suggest improvement(s).